ocs_ci/deployment/deployment.py

```diff
@@ -2053,6 +2056,33 @@ def get_and_apply_icsp_from_catalog(image, apply=True, insecure=False):
     return icsp_file_dest_location
 
 
+def get_latest_acm_tag_unreleased(version):
```

Comment:
Please move it out of deployment.py module to ocs_ci/utility/utils.py where we have also get_latest_ds_olm_tag .

Reply:
I tried consuming get_latest_ds_olm_tag but the function did not worked for me